### PR TITLE
Document scaling

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <p.loffay@gmail.com>

### CODE REVIEW

**Comment by pavolloffay** [comment]
@EinKrebs  could you please review or send it to folks from clickhouse team?

File: guide-sharding-and-replication.md [hunk 1]
```diff
@@ -127,6 +127,10 @@ kind: ClickHouseInstallation
 metadata:
   name: jaeger
 spec:
+  defaults:
```
Comment (pavolloffay) [inline guide-sharding-and-replication.md:130]: change to this CR is not related to this PR but it fixes using of persistent volumes.

**Comment by EinKrebs** [comment]
@pavolloffay I asked for somebody to review this.

**Comment by pavolloffay** [comment]
Feel free to comment I will address comment in follow-up PR.

**Pull request merged**